external-import/misp/src/misp.py

```diff
@@ -1879,9 +1879,20 @@ def prepare_elements(self, galaxies, tags, author, markings):
                     threats = self.helper.api.stix_domain_object.list(
                         types=["Intrusion-Set", "Malware", "Tool", "Attack-Pattern"],
                         filters={
```

Comment:
You can write this filter in a simpler way : 
filters = {
   "mode": "and",
   "filters": [
      {"key": ["name", "xi_mitre_id", "alias"], "values": [tag_value]}
   ],
   "filterGroups": [],
}